news/feat-add-lb-strategy.md

```diff
@@ -0,0 +1 @@
+    Add support for saving only the lower bound `x <= VERSION` when adding dependencies.
```

Comment:
I think you meant `>=`?
```suggestion
    Add support for saving only the lower bound `x >= VERSION` when adding dependencies.
```

Reply:
Oh no... yes! Thanks for the quick catch.